examples/gallery/plot/scatter.py

```diff
@@ -21,19 +21,19 @@
     projection="X10c/10c",
     frame=["xa0.2fg", "ya0.2fg", "WSrt"],
 )
-for color in ["blue", "orange", "green"]:
+for color, style in [("gray73", "c"), ("darkorange", "s"), ("slateblue", "t")]:
     x, y = np.random.rand(2, n)  # random X and Y data in [0,1]
     sizes = np.random.rand(n) * 0.5  # random size [0,0.5], in cm
     # plot data points as circles (style="c"), with different sizes
     fig.plot(
         x=x,
         y=y,
-        style="c",
+        style=style,
         sizes=sizes,
         color=color,
         # Set the legend label,
-        # and set the circle size to be 0.25 cm (+S0.25c) in legend
-        label=f"{color}+S0.25c",
+        # and set the style size to be 0.25 cm (+S0.25{style}) in legend
+        label=f"{color}+S0.25{style}",
```

Comment:
See the warning at https://pygmt-ln1ssu8ap-gmt.vercel.app/gallery/plot/scatter.html#sphx-glr-gallery-plot-scatter-py:
```
plot [WARNING]: Length <unit> s not supported - revert to default unit [cm]
plot [WARNING]: Length <unit> t not supported - revert to default unit [cm]
```

`0.25c` means 0.25 cm, so we shouldn't use `style` as the unit. If we'd like to change the size for each symbol in the legend, it'd better to also add it in the for loop. But I feel `0.25c` is okay for those three symbols.
```suggestion
        label=f"{color}+S0.25c",
```

Reply:
Thank you so much, and I'm sorry, I didn't know that, I thought it was the symbol to show, not the size in cm. Appreciated.